Source/AwsCommonRuntimeKit/crt/Hash.swift

```diff
@@ -48,7 +47,7 @@ extension Data {
     /// Computes the sha1 hash over data.
     /// - Parameter truncate: If you specify truncate something other than 0, the output will be truncated to that number of bytes. For
     /// example, if you want a SHA1 digest as the first 10 bytes, set truncate to 10.
-    public func sha1(truncate: Int = 0) throws -> Data {
+    public func compteSHA1(truncate: Int = 0) throws -> Data {
```

Comment:
spelling error -- should be `computeSHA1`

Reply:
Thanks, fixed.